### DISCUSSION TITLE
German translation

### DISCUSSION THREAD

**Discussion by oliver303** [community/community#22140] [New to GitHub]
<p>Hi!<br>
On Chrome for Android you support the german language. But there’s a translation issue. <code>sign up</code> and <code>sign in</code> can be both translated into german with the verb <code>anmelden</code>. This is what you did. It’s translated correctly. No issue with that! :slightly_smiling_face:</p>
<p>But you’ll end up with two buttons saying <code>Anmelden</code>. Just they’re formatted differently. That’s confusing! :thinking:</p>
<p>You should use the german verb <code>registrieren</code> if you mean <em>sign up</em> in the sense of <em>creating a new account</em>  and the german verb <code>anmelden</code> if you mean <em>sign in</em> in the sense of <em>logging on</em>.</p>
<p>I never noticed this issue because I’m using the english interface. But my girlfriend prefers to use the german translation.</p>
<p>Greetings from Vienna<br>
Oliver</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi Oliver! :wave: Welcome to the Community!</p>
<p>Do you mean the GitHub web interface? We only provide an English version, so if you’re seeing a translated version , it’s due to a Chrome plugin you’re using, or some other auto-translation your end.</p>
<p>I’m afraid we have no control over how such third party services translate the site, sorry!</p>

**Comment by oliver303** [comment]
<p>OK! I see.<br>
So close this issue,please.<br>
Regards Oliver</p>

**Comment by oliver303** [comment]
<p>Hi!<br>
Yes I mean the web interface. Just to clarify.</p>
